config/reports/lpdcBestuurseenheidReport.js

```diff
@@ -0,0 +1,36 @@
+import {generateReportFromData} from '../helpers.js'
+import { querySudo as query } from '@lblod/mu-auth-sudo';
+
+export default {
```

Comment:
How did you test this? 
It seems missing: https://github.com/lblod/app-digitaal-loket/blob/master/config/reports/index.js

The stack has also a frontend app: dashboard, you should be able to check it there